src/index.ts

```diff
@@ -22,5 +22,5 @@ export {
     UserMetadataField,
     WebAuthnDevices,
 } from './generated';
-
-export default Passage;
+import { Passage } from './classes/Passage';
+export default Passage;
```

Comment:
`./classes/Passage` is being exported and imported. Is that correct? If so, why?

Reply:
Yes, the `export { Passage } from './classes/Passage'` exports the Passage class as a named member so when using the package you can do `import { Passage } from '@passageidentity/passage-node'`

The import statement is because I need to be able to import it first to be able to write `export default Passage` for backwards compatibility.

Reply:
In the next major release I intend to remove 
``` 
import { Passage } from './classes/Passage';
export default Passage;
```